import "EquipmentInfo.proto";

message DseUpdateEquipmentInfo
{
	repeated EquipmentInfo		infos = 1;
}